test/error.js

```diff
@@ -3,7 +3,7 @@ const assert = require('assert')
 
 describe('error handling', () => {
   it('throws an error when passed a bad language in the function', () => {
-    function humanizingWith (options) {
+    function humanizingWith(options) {
```

Comment:
I'm not sure why this was changed. Could you revert the spacing difference?

```suggestion
    function humanizingWith (options) {
```